app/api/models.py

```diff
@@ -182,6 +211,7 @@ class Meta:
 
 class Document(models.Model):
     text = models.TextField()
+    extra_text = models.TextField(blank=True)
```

Comment:
Wondering if instead we could/should use the `meta` field to store this extra data. To make working with `meta` easier, we could consider adding a computed property like this:

```py
@property
def metadata(self):
    return json.loads(self.meta)
```

Reply:
From my understanding, `meta` was related to external information such as `external_id`. Here, this property is core to some task that may require more than one text. It could fit in the meta I believe, but made more sense to put it as a property to me. your call.